.gitignore

```diff
@@ -1,396 +1 @@
-# Created by https://www.toptal.com/developers/gitignore/api/intellij+all,intellij+iml,python,visualstudiocode,macos,linux,venv
```

Comment:
Could we please keep the .gitignore and append to it, some developers use intellij (me) and others vscode and those come with generated files that shouldn't be pushed to the main repo

Reply:
Yes, sorry. It was overwritten at some point. Please notice that `[Ss]cripts` needs to be commented out, as there is a folder called `scripts` that is relevant to the snap